### DISCUSSION TITLE
Weekly Dev Chat

### DISCUSSION THREAD

**Discussion by popalexandrudan** [sunflower-land/sunflower-land#2295] [Ideas]
I think that the weekly dev chat as they are at the moment are very informative, but could be better.
At the moment, most of them lack video content, and i think that hurts the player engagement.

Some random thoughts on how they could be improved :

- sharing a video with a power point
- showing the official docs
- random SFL gameplay (last week could have just been live feed from the plaza...its interesting to see you guys taking about it, but not being able to see it feels wrong)

**Comment by adamhannigan** [comment]
@popalexandrudan We are limited by Discords limit of 50 people when sharing a screen.

Would you prefer this on Twitch instead?

**Reply by popalexandrudan** [reply]
i think twitch would definitely be a better option, and you would have video footage to share on the youtube weekely videos as well.  Of course this is something subjective, so i would love to hear other people opinions.

**Reply by IvanSkazkin** [reply]
@adamhannigan I don't know English. But in my country there is an online language translator for YouTube. If you posted your twitch recordings on YouTube somewhere, I could watch them.
